index.html

```diff
@@ -0,0 +1,139 @@
+    <script async src="https://www.googletagmanager.com/gtag/js?id=G-ZYTLN5HPEC"></script>
+    <script>
+      window.dataLayer = window.dataLayer || [];
+      function gtag(){dataLayer.push(arguments);}
+      gtag('js', new Date());
+    
+      gtag('config', 'G-ZYTLN5HPEC');
+    </script>
+    <title>CIO - CTO Global Summit</title>
+</head>
+<body>
+    <header class="top-header">
+        <div class="header-content">
+            <ul class="header-list">
+                <li><img src="images/facebook.png" alt="Facebook"></li>
+                <li><img src="images/twitter.png" alt="twitter"></li>
+                <li>English</li>
+                <li>My Page</li>
+                <li>Logout</li>
+            </ul>
+        </div>
+    </header>
+
+<nav class="navigation">
+    <div class="logo">
+        <a href="index.html">
+        <img class="logo-img" src="images/Logo-Light-2.png" alt="Logo"></a>
+    </div>
+    <!-- mobile menu -->
+    <div class="menumobile">
+        <nav class="navbar-mobile">
+            <div class="hamburguer-menu">
+                <div class="line line-1"></div>
+                <div class="line line-2"></div>
+                <div class="line line-3"></div>
+            </div>
+            <ul class="hamb-nav-list">
+                <li id="menu-item1" class="hamb-nav-item"><a href="#" target="_self" class="nav-link">About</a></li>
+                <li id="menu-item2" class="hamb-nav-item"><a href="#" target="_self" class="nav-link">Program</a></li>
+                <li id="menu-item3" class="hamb-nav-item"><a href="#" target="_self" class="nav-link">Join</a></li>
+                <li id="menu-item4" class="hamb-nav-item"><a href="#" target="_self" class="nav-link">Sponsor</a></li>
+                <li id="menu-item5" class="hamb-nav-item"><a href="#" target="_self" class="nav-link">News</a></li>
+            </ul>
+        </nav>
+    </div>
+    <!-- end mobile menu -->   
+    <div class="menu">
+        <ul class="menu-nav">
+            <li><a href="about.html">About</a></li>
+            <li>Program</li>
+            <li>Join</li>
+            <li>Sponsor</li>
+            <li>News</li>
+        </ul>
+    </div>
+    <div class="btn-campain"></div>
+</nav>
+   
+<section class="hero">
+    <img class="desktop-img" src="images/main_big.png" alt="">
+    <img class="mobile-img" src="images/mobile_intro_bg.png" alt="">
+    <div class="hero-inner">
+        <p class="call-title">The World CIO 200 Summit presents:</p>
+        <h1 class="title">C-LEVELS & MANAGERS <br/>GLOBAL SUMMIT 2021</h1>
+        <p class="hero-paragraphy">A joyful celebration believing in the value of opperness and sharing, creating a positive change with people from over 80 countries is talking place, in August, in Brazil</p>
+        <br>
+        <h3 class="date-title">2021.08.30(MON) ~ 31(TUE)</h3>
+        <p class="location">@ Ouro Minas Palace Hotel</p>
+    </div>
+</section>
```

Comment:
- [ ] The overlapping issue still exists. You **should avoid positioning your elements absolutely.** It takes elements out of your document flow. This could lead to bad situations in the future and is considered bad practice if done too much. You should use margin/padding to position your elements. I think the alignment issue is also related to this because when you absolutely the result will be different according to screen size.
![Screenshot 2021-08-14 at 01-36-56 CIO - CTO Global Summit](https://user-images.githubusercontent.com/1642014/129424822-d33906d4-3349-42a8-9a0d-fd667aa3399f.png)